### PR TITLE
Account Page documentation

### PR DESCRIPTION
**Pull request by nilssonemma**

### CODE REVIEW

File: docs/integrate/merchant/account-page.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Account Page
+
+Intergiro Account helps you to manage payments for customers with which you have an ongoing relationship. A personal account page is provided to all your customers, to make it easy to update payment and contact information, and to provide easy access to all orders associated to the account.
+
+## Website integration
+ The account page can be integrated into your website as an embedded page. The `<intergiro-account>` component can be used to access the entire account page, or to access the single components of the page, such as the payment methods, order history or subscriptions.
+ 
+To integrate the account page to your website, a unique [account key](#account-api-key) is needed for each customer. Account keys are valid for three days until they expire and should thereafter be renewed.
```
Comment (marcushanses) [inline docs/integrate/merchant/account-page.md:8]: Account keys -> (account key)(account-api-key)s ??

File: docs/integrate/merchant/account-page.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<intergiro-account api-key="<your-account-key>"></intergiro-account>
+```
+Single component example implementation. Possible component identifiers are `"method"`, `"orders"` or `"subscription"`. 
+```html
+<intergiro-account api-key="<your-account-key>" component="<component-identifier>"></intergiro-account>
+```
+
+
+## Account API key
+In order to create an Account API key, an account id is needed together with a private key for authorization.
```
Comment (marcushanses) [inline docs/integrate/merchant/account-page.md:21]: Rewrite?

File: docs/integrate/merchant/account-page.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+A successful response will return an Account API key.
+
+## Customized theming
+Customized theming can be set to the account page by specifying specific styling attributes. This is done by entering a stringified JSON object into the `cosmetic` attribute. All available styling options are seen in the figure below.
+
+```json
+ cosmetic={
+    "fontFamily": "Arial, Verdana, Helvetica",
+    "header": {"background": "#fff", "color": "black"},
```
Comment (marcushanses) [inline docs/integrate/merchant/account-page.md:41]: remove a "}".
Only use RBG.
Double check the colors.

File: docs/integrate/merchant/account-page.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+Example implementation with a customized theme: 
+```html
+<intergiro-account api-key="<your-account-key>" cosmetic='{
+    "fontFamily": "Arial, Verdana, Helvetica",
+    "header": {"background": "#fff", "color": "black"},
+    "body":{"background":"227,227,227","color":"0,0,0"},
+    "success":{"background":"81,196,115","color":"255,255,255"}}
+  }'></intergiro-account>
```
Comment (marcushanses) [inline docs/integrate/merchant/account-page.md:59]: remove a "}".
Only use RBG.
Double check the colors.

**Review by marcushanses — CHANGES_REQUESTED** [review]

File: docs/integrate/merchant/account-page.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# Account Page
+
+Intergiro Account helps you to manage payments for customers with which you have an ongoing relationship. A personal account page is provided to all your customers, to make it easy to update payment and contact information, and to provide easy access to all orders associated to the account.
```
Comment (robinnedergard) [inline docs/integrate/merchant/account-page.md:3]: Ongoing relationship?

File: docs/integrate/merchant/account-page.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Website integration
+ The account page can be integrated into your website as an embedded page. The `<intergiro-account>` component can be used to access the entire account page, or to access the single components of the page, such as the payment methods, order history or subscriptions.
```
Comment (robinnedergard) [inline docs/integrate/merchant/account-page.md:6]: or to access ~the single~ _individual_ components of the page

File: docs/integrate/merchant/account-page.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+```json
+ cosmetic={
+    "fontFamily":"Open Sans, sans-serif",
+    "header":{"background":"86,86,86","color":"255,255,255"},
+    "body":{"background":"227,227,227","color":"0,0,0"},
+    "success":{"background":"109, 165, 59","color":"255,255,255"}
+  }
+```
+
+Note: Colors are specified using RGB comma seperated string e.g. "0,0,255" for blue.
```
Comment (robinnedergard) [inline docs/integrate/merchant/account-page.md:47]: ~seperated~ _separated_

**Review by robinnedergard — CHANGES_REQUESTED** [review]

File: docs/integrate/merchant/account-page.md (hunk 5 above)
Comment (robinnedergard) [reply docs/integrate/merchant/account-page.md:3]: Intergiro Account is a tool for customers to manage payment methods, orders and subscriptions.

File: docs/integrate/merchant/account-page.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+<intergiro-account api-key="<your-account-key>"></intergiro-account>
+```
+Single component example implementation. Possible component identifiers are `"method"`, `"orders"` or `"subscription"`. 
+```html
+<intergiro-account api-key="<your-account-key>" component="<component-identifier>"></intergiro-account>
+```
+
+
+## Account API key
+In order to create an Account API key, an account id and a private key is needed.
```
Comment (robinnedergard) [inline docs/integrate/merchant/account-page.md:21]: Depending on the private key used to create the account keys, certain features may be explicitly disabled or enabled.

File: docs/integrate/merchant/account-page.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
+# Account Page
+
+Intergiro Account is a tool for customers to manage payment methods, orders and subscriptions. A personal account page is provided to all your customers, to make it easy to update payment and contact information, and to provide easy access to all orders associated to the account.
```
Comment (marcushanses) [inline docs/integrate/merchant/account-page.md:3]: Intergiro Account is a  personal account page that is provided to all your customers, to make it easy to update payment and contact information, and to provide easy access to all orders associated to the account.

**Review by marcushanses — APPROVED** [review]

**Review by robinnedergard — APPROVED** [review]

**Review by florianebmeier — APPROVED** [review]

File: docs/integrate/merchant/account-page.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## Account API key
+In order to create an Account API key, an account id and a private key is needed. Depending on the private key used to create the account keys, certain features may be explicitly disabled or enabled. 
+
+Example request:
+``` {1}
+POST /account/<accountId>
+
+Host: api.payfunc.com
```
Comment (euqen) [inline docs/integrate/merchant/account-page.md:27]: lets change to merchant.intergiro.com even if it's not available yet

File: docs/.vuepress/config.js [hunk 11]
```diff
@@ -57,6 +58,7 @@ module.exports = {
       '/integrate/card-programmes/': getIntegrateSidebar(),
       '/integrate/embedded-finance/': getIntegrateSidebar(),
       '/integrate/acquiring/': getIntegrateSidebar(),
+      '/integrate/merchant/': getIntegrateSidebar(),
```
Comment (euqen) [inline docs/.vuepress/config.js:61]: I guess this is for our merchants not for psps, right? If so please move under /merchant do not keep it under /integrate please

**Pull request merged**